vendor/impl/rocksdb_block_storage_factory.cpp

```diff
@@ -0,0 +1,7 @@
+#include "vendor/impl/rocksdb_block_storage.hpp"
+
+#include "vendor/impl/rocksdb_block_storage_factory.hpp"
+
+std::unique_ptr<iroha::ametsuchi::BlockStorage> RocksdbBlockStorageFactory::create() {
```

Comment:
Check the path provider approach used in `FlatFileBlockStorageFactory`.

Reply:
Done.